guest-test/guest.test_launcher.sh

```diff
@@ -282,6 +284,7 @@ if ! guest_kernel_check; then
 		pkill "${VM_TYPE}vm_$PORT"
 		die "TEST TIMEOUT!!!!!!!!!!!!"
 	elif [ "$GCOV" == "off" ] && [ "$EXEC_FLAG" -eq 1 ]; then
+    pkill "${VM_TYPE}vm_$PORT"
```

Comment:
Hi Yongyu,
You mixed 'tabs' and 'spaces' for the indentation in the whole file.
Would you like to fix it?

Reply:
fixed in new commit